scripts/lotus-daemon.service

```diff
@@ -1,19 +1,17 @@
 [Unit]
 Description=Lotus Daemon
-After=network-online.target
-Requires=network-online.target
+After=network.target
 
 [Service]
-Environment=GOLOG_FILE="/var/log/lotus/daemon.log"
-Environment=GOLOG_LOG_FMT="json"
+User=fc
+Group=fc
+ExecStartPre=/usr/local/bin/lotus-init.sh
```

Comment:
Can we just pass `--import-chain` to the daemon? I believe that should import iff not already initialized.

Reply:
--import-chain always imports

Reply:
import-chain will force a validation back to ts-0 
https://github.com/filecoin-project/lotus/blob/master/cmd/lotus/daemon.go#L529

running --import-chain against minimal_finality_stateroots does result in a validation error. Although aside from this, I don't think there is a difference between the two import procedures.

Reply:
Ah... That's, unfortunate.